sql/plan.go

```diff
@@ -80,6 +80,8 @@ func (p *planner) makePlan(stmt parser.Statement) (planNode, error) {
 		return p.Select(n)
 	case *parser.Set:
 		return p.Set(n)
+	case *parser.SetTransaction:
+		return p.SetTransaction(n)
```

Comment:
In postgres, the `SET TRANSACTION ISOLATION LEVEL` statement sets the isolation level for the current transaction. There is a `SET SESSION CHARACTERISTICS AS TRANSACTION` statement for setting session defaults for transactions, but this change isn't implementing that.